Source/WebKit/NetworkProcess/cocoa/NetworkTaskCocoa.mm

```diff
@@ -155,7 +166,11 @@ static bool shouldCapCookieExpiryForThirdPartyIPAddress(const WebCore::IPAddress
             if (!remoteAddress)
                 return cookiesSetInResponse;
 
-            if (shouldCapCookieExpiryForThirdPartyIPAddress(*remoteAddress, *firstPartyAddress)) {
+            auto needsThirdPartyIPAddressQuirk = [] (const URL& requestURL) {
```

Comment:
Please consider adding a comment to the effect of "under google.com" here to not cause confusion for someone casually reading the code and thinking that accounts.google.com has some general exception. As I understand it, this code only gets called in the "subdomain under first party" case so it's just about being very clear in the code.

Reply:
We should also note that this rules out any subdomain of accounts.google.com, which I think is fine.

Reply:
Sounds good — will leave a clarifying comment.

Thanks for the reviews!